static/styles/left_sidebar.css

```diff
@@ -235,7 +235,7 @@ li.active-sub-filter {
 
     .count {
         margin-right: 20px;
-        margin-top: 2px;
+        margin-top: 1px;
```

Comment:
Isn't this going to change the centering for all the unread counts in that part of the screen?

`manage.py mark_all_messages_unread` may be useful for testing.

Reply:
before:

![GifMaker_20210414152810053](https://user-images.githubusercontent.com/39272041/114692388-33ee9280-9d36-11eb-9d69-2f114cabf69d.gif)

after:
![GifMaker_20210414152622441](https://user-images.githubusercontent.com/39272041/114692441-3e109100-9d36-11eb-9704-9f83f4624938.gif)


This change did not much effect the all_unread_counts, but it is useful to make star count boundry to not touch the box.